compiler/src/test/java/tiamat/compiler/PrefsProcessorTest.java

```diff
@@ -18,12 +18,13 @@
 
     @Test
     public void tableNameNotDefined() {
```

Comment:
actual error message is here.

https://gist.github.com/yshrsmz/97e8585e5150a790f226a1f5560c5401

basically it complains about unresolvable reference.

But as I stated in PR, we can't import "library" module in "compiler" module.

Any ideas how to solve this issue?